applications/minotari_ledger_wallet/wallet/src/handlers/get_script_offset.rs

```diff
@@ -199,6 +202,7 @@ pub fn handler_get_script_offset(
     comm.append(&[RESPONSE_VERSION]); // version
     comm.append(&script_offset.to_vec());
     offset_ctx.reset();
+    offset_ctx.finished = true;
```

Comment:
Was this to do with your state issue?
What exactly is the bool solving, I think I'm overlooking its purpose.

Reply:
an attempt at getting the UI working, good catch this needs to be removed